src/services/http/connector.js

```diff
@@ -0,0 +1,203 @@
+'use strict'
+
+module.exports = [
+  {
+    pin: 'role:auth,cmd:*',
+    prefix: 'auth',
+    map: {
+      /**
+         *
+         * @api {POST} /auth/login adminLogin
+         * @apiName adminLogin
+         * @apiGroup Auth
+         * @apiVersion  1.0.0
+         *
+         *
+         * @apiParam  {String} username admin username
+         *
+         * @apiParam  {String} password admin password
+         *
+         *
+         * @apiParamExample  {type} Request-Example:
+         * {
+         *     username : "justice"
+         *     password : "1234"
+         * }
+         *
+         *
+         * @apiSuccessExample {type} Success-Response:
+         *
+         * {
+         *       "data": {
+         *           "token": "eyJhbGciOiJIUzI1N............"
+         *       },
+         *       "status": 200
+         *
+         *
+         */
+      login: {
+        POST: true
+      },
+      /**
+         *
+         * @api {POST} /auth/signup adminSignup
+         * @apiName adminSignup
+         * @apiGroup Auth
+         * @apiVersion  1.0.0
+         *
+         *
+         * @apiParam  {String} username admin username
+         *
+         * @apiParam  {String} password admin password
+         *
+         *
+         * @apiParamExample  {type} Request-Example:
+         * {
+         *     username : "justice"
+         *     password : "1234"
+         * }
+         *
+         *
+         * @apiSuccessExample {type} Success-Response:
+         *
+         * {
+         *       "data": {
+         *           "token": "eyJhbGciOiJIUzI1N............"
+         *       },
+         *       "status": 200
+         *
+         *
+         */
+      signup: {
+        POST: true
+      }
+    }
+  },
+  {
+    pin: 'role:user,cmd:*',
+    prefix: '/user',
+    map: {
+      /**
+         * @api {get} /user/:id GetUser
+         * @apiName GetUser
+         * @apiGroup User
+         * @apiVersion  1.0.0
+         *
+         * @apiParam {ObjectID} id Users unique ID.
+         *
+         *
+         * @apiSuccessExample Success-Response:
+         *     HTTP/1.1 200 OK
+         *       {
+         *       "data": {
+         *           "_id": "5c99b8dce69885448104360d",
+         *           "name": "jaks",
+         *           "dob": "12/12/1990",
+         *           "address": "ashdjakhsdk",
+         *           "description": null
+         *       },
+         *       "status": 200
```

Comment:
could you let me know the value of including the `status` in the response body considering we already have http statusCode?

Reply:
My choice to include the status in the response comes from error handlin. Its easy to understand when the server returns an error of 400 stating an error, but if within the organization it might be possible that specific custom codes might require different actions. For example, we could have an error code 4443 that corresponds to a specific error caused by improper usage of a specific component of the api which would be more descriptive than a normal 400